coq-nix-toolbox.nix.mustache

```diff
@@ -0,0 +1,4 @@
+# This file was generated from meta.yml, please do not edit manually.
+# Follow the instructions on https://github.com/coq-community/templates to regenerate.
+# However, it not handled by mustache, but by a special eval command
+\"$(git ls-remote https://github.com/coq-community/coq-nix-toolbox refs/heads/master | cut -f1)\"
```

Comment:
I see the point of this change: you always get the latest version of the toolbox when you run the templates. But isn't it problematic? E.g., what happens if I didn't `git pull` the latest changes and the generated templates are outdated?

Discussed live. Conclusion: use a variable in `meta.yml` to not pin instead.

Reply:
Done! Now pinning to last is activated by:
```
nix-default:
  ref: master
```
(and you can change the ref, the owner or the url at will)